### DISCUSSION TITLE
What would be the equivalent of WalletConnect client?

### DISCUSSION THREAD

**Discussion by asenrusev** [WalletConnect/walletconnect-monorepo#2826] [Developer Support] [answered]
Hello,

We're heavily relying on this [client](https://docs.walletconnect.com/1.0/quick-start/wallets/react-native), what would be the best way to replace it for v2?

We're using the following properties:
- connect
- connected
- createSession
- killSession
- signPersonalMessage
as well as the following events:
- "connect",
- "call_request_sent",
- "session_update",
- "disconnect",

Thank you!

**Comment by crypblizz8** [comment, marked as answer]
Hey there, this one: https://github.com/WalletConnect/web3modal-react-native

It uses Universal Provider + we have tested the ethers methods to do the above events / methods. Feel free to check out the example folder

**Reply by asenrusev** [reply]
That one is deprecated. Would it be okay to use the offered suggestion?
